packages/web/src/components/nav/mobile/NavBar.tsx

```diff
@@ -84,11 +91,17 @@ const NavBar = ({
   useEffect(() => {
     const splitPath = pathname.split('/')
     const isSearch = splitPath.length > 1 && splitPath[1] === 'search'
-    if (!isSearch) {
-      setIsSearching(false)
-    }
+    setIsSearching(isSearch)
   }, [pathname])
 
+  const handleOpenSearch = useCallback(() => {
+    if (isSearchV2Enabled) {
+      history.push(`/search`)
```

Comment:
do we want to dispatch the push?

Reply:
Yup, making that change now!

Reply:
Ah actually even if I don't dispatch it a router location change action still occurs. I noticed some other places we just use history.push so just double checked